CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # Changelog
 
-> [!IMPORTANT]  
-> If you are upgrading to the `8.x` versions of the SDK from `7.x` or below, make sure you follow our
+> [!IMPORTANT] If you are upgrading to the `8.x` versions of the SDK from `7.x` or below, make sure you follow our
```

Comment:
I also have this. Super annoying but I think if you don't change it back, github will render it as

> [!IMPORTANT] If you are upgrading to the `8.x` versions of the SDK from `7.x` or below, make sure you follow our

instead of

> [!IMPORTANT]
> If you are upgrading to the `8.x` versions of the SDK from `7.x` or below, make sure you follow our

Reply:
oh nevermind this doesn't show in the releases tab

Reply:
yeah but it f*s up the rendered markdown file. I ignored auto format